sprockets/mixins/metrics/statsd.py

```diff
@@ -174,6 +175,12 @@ def send(self, path, value, metric_type):
         except (OSError, socket.error) as error:  # pragma: nocover
             LOGGER.exception('Error sending statsd metric: %s', error)
 
+    def _build_udp_or_tcp_message(self, msg):
+        if self._tcp is False:
+            return msg
+
+        return msg + "\n"
+
```

Comment:
Just make the format an attribute instead of adding another function call and string concatenation to each metric send.  You can set the format string (eg, `'{path}:{value}|{type}'` or `'{path}:{value}|{type}\n'`) in the [initializer](https://github.com/ezhidblr/sprockets.mixins.metrics/blob/6fd2cf40352517839b854b627f4b04c3a6551d19/sprockets/mixins/metrics/statsd.py#L124-L131) and change the send code to:
```python
msg = self._payload_format.format(
   path=self._build_path(path, metric_type),
   value=value, type=metric_type)
```